README.md

```diff
@@ -19,7 +19,7 @@ Directories (read or read-write)
 > B. Generates checksum files and compares checksum files.
 
 ### input types:
-The validation action can be specified using the -t tag.
+The validation action can be specified using the -t tag. If not specified, it defaults to "file-input".
```

Comment:
Use backtickts instead of double quotes as you did in `CHANGELOG` (`-t` and `file-input`)

L67 looks outdated but maybe we can create a new PR to update README?